OutOfSchool/OutOfSchool.WebApi/Services/ProviderAdminService.cs

```diff
@@ -498,6 +498,35 @@ public async Task<IEnumerable<string>> GetProviderDeputiesIds(Guid providerId)
         return providersDeputies.Select(d => d.UserId);
     }
 
+    /// <inheritdoc/>
+    public async Task<ProviderAdminDto> GetFullProviderAdmin(ProviderAdminProviderRelationDto providerAdmin)
```

Comment:
why do you need to call get by id in controller then pass this object down here and call this method?:) can't it all be do inside here? Also, `GetByFilter(Expression<Func<TEntity, bool>> predicate, string includeProperties = "")` has `includeProperties` so you can use this method to both get by id and include other entities via join

Reply:
I described a reason in a comment above.